meta-lmp-base/conf/distro/lmp-wayland.conf

```diff
@@ -5,3 +5,4 @@ DISTROOVERRIDES = "lmp:lmp-wayland"
 DISTRO_NAME = "Linux-microPlatform Wayland"
 
 DISTRO_FEATURES:append = " wayland opengl vulkan"
+IMAGE_INSTALL:append:lmp-wayland = " wayland-cdi"
```

Comment:
I would probably move this to meta-lmp-base/recipes-samples/images/lmp-feature-docker.inc, and have it conditional to wayland (via DISTRO_FEATURES):

`${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland-cdi, '', d)}`

Reply:
force push update https://github.com/foundriesio/meta-lmp/compare/0080ebe9c6f97c0fd5313c45d3b959ff5129c92c..3d1aeb0e60dd1fb9d0fd1e18e42c66d1497ac975